Cargo.toml

```diff
@@ -17,7 +18,7 @@ arrayref = "0.3"
 byteorder = "1.3.1"
 data-encoding = "2.1"
 multibase = "0.9.1"
-multihash = { version = "0.17", default-features = false, features = ["std", "multihash-impl", "identity"] }
+multihash = { version = "0.18", default-features = false, features = ["std", "multihash-impl", "identity"] }
```

Comment:
We will be doing another breaking change with https://github.com/multiformats/rust-multiaddr/pull/70. https://github.com/multiformats/rust-multiaddr/pull/70 is needed among other things for better instrumentation of the IPFS network.

Given that we do a breaking change anyways, we might as well pack as much as possible into it, i.e. `multihash` `v0.18`.

Reply:
https://github.com/multiformats/rust-multiaddr/issues/73 is also a breaking change, can we include that as-well? I don't have permissions to create a milestone in this repository but it might be worthwhile creating one so we can plan these things :)